src/main/java/com/beerjournal/breweriana/item/rating/RatingController.java

```diff
@@ -0,0 +1,37 @@
+package com.beerjournal.breweriana.item.rating;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/ratings")
+@RequiredArgsConstructor
+public class RatingController {
+
+    private final RatingService ratingService;
+
+    @GetMapping("item/{itemId}")
```

Comment:
Not entirely, as rating for now is not something that item owns. And present paths are shorter.  
If these arguments don't convince you, I will change it, as you are The Repo Owner :P